src/components/Card/Card.tsx

```diff
@@ -26,42 +27,50 @@ const iconMap: { [key: string]: any } = {
 }
 
 const Card = ({plant}: CardProps) => {
-    const [{ isDragging }, dragRef] = useDrag({
+    const [dragging, setIsDragging] = useState<boolean>(false)
+
+    const [{ isDragging }, dragRef, preview] = useDrag({
         type: 'plant',
         item: { plant },
         collect: (monitor) => ({
             isDragging: monitor.isDragging()
         })
     })
 
+    useEffect(() => {
+        setIsDragging(!dragging)
+        // eslint-disable-next-line
+    }, [isDragging])
```

Comment:
What exactly is this trying to do? I don't see `dragging` actually used anywhere?

Reply:
Ah sorry, I just read your comment:
> Try dragging the same card over and over again. I had to implement the useEffect because the DragPreviewImage would lose its connection after the initial drag. Forcing a re-rerender as our isDragging monitor gets updated seems to have fixed it, but please ensure.

I have no idea why it loses its connection, but perhaps a better way to re-render the Drag preview image is to do this:
```
{!isDragging && (
  <DragPreviewImage connect={preview} src={`${process.env.PUBLIC_URL}/images/plant.png`} />
)}
```

This is cleaner and should also fix the issue so that it works on every subsequent drag too. Though again, unsure why it loses the connection in the first place 🤷

Reply:
I just tested this on my laptop and it works!